NavigationSystem/Tests/IntegrationTests/CANAISTest.cpp

```diff
@@ -0,0 +1,70 @@
+// #pragma once
```

Comment:
Add a little description on the file : purpose and your comments. 
Also I don't understand how you tests. Need to see that with you